android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MatchingEngineHelper.java

```diff
@@ -75,7 +76,7 @@ public MatchingEngineHelper(Context context, String hostname, String carrierName
         mHostname = hostname;
         mCarrierName = carrierName;
         mMatchingEngine = new MatchingEngine(mContext);
-        mMatchingEngine.setHost(mHostname);
+        // mMatchingEngine.setHost(mHostname);

```

Comment:
Is there a reason for commenting this out? I remember needing to add it for some reason, but I can't remember why.

Reply:
Actually wondering if we just need to remove it from the SDK entirely since DME is intended to be quite ephemeral now.

Reply:
setHost was either gone or not public in 1.4.19